detect/synthetic-monitoring/playwright-checks/overview.mdx

```diff
@@ -5,65 +5,54 @@ description: 'Run your Playwright end-to-end test suite as synthetic production
 tags: ['synthetic-monitoring', 'playwright-checks']
 ---
 
-<Tip>
-**Monitoring as Code**: Learn more about the [Playwright Check Suite Construct](/constructs/playwright-check).
-</Tip>
-
 ## What are Playwright Check Suites?
 
-Playwright Check Suites enable you to run entire Playwright end-to-end test suites, Playwright projects or tagged tests as pre-production tests and global production monitors, without rewriting your tests and configuration.
+Playwright Check Suites turn your Playwright end-to-end tests into production monitors. Run entire test suites, specific projects, or tagged tests globally without modifying your existing code or configuration.
 
-Unlike [Browser Checks](detect/synthetic-monitoring/browser-checks/overview), Playwright Check Suites are not limited to a single test spec and our preinstalled runtime dependencies. Playwright Check Suites enable you to run Playwright projects complete with all of your required dependencies, private packages, storage, data, and more, and enable you to detect issues with your applications or services even in the most complex scenarios.
+<Warning>
+**Known limitations**: Playwright Check Suites do not currently support monorepo workspaces for dependency installation.
+Each check run has a maximum execution time of 30 minutes.
+</Warning>
 
 <Frame>
   ![Playwright Check Suites test result](/images/pwn-test-result.png)
 </Frame>
 
-**Playwright Check Suites are the native way to run Playwright in production** and support the full Playwright API and ecosystem, meaning you can use your existing tests and `playwright.config.ts` files as is.
-
-**Playwright Check Suites are perfect for:**
-- Converting existing E2E tests into monitoring
-- Testing complex user workflows across multiple browsers
-- Monitoring critical business processes
-- Validating application functionality after deployments
+**Use Playwright Check Suites to:**
+- Turn E2E tests into global synthetic monitors
+- Monitor complex user workflows across browsers and regions
+- Detect issues in critical business processes
+- Verify functionality after deployments
 
 ## How Playwright Check Suites Work
 
 <Steps>
-<Step title="1. Test Discovery">
-Playwright Check Suites analyze your Playwright configuration and test files to understand your existing test structure.
+<Step title="1. Select tests">
+Choose which tests to monitor using Playwright projects or tags from your existing configuration.
 </Step>
+<Step title="3. Choose frequency and locations">
```

Comment:
step 3 comes before 2 😁

Reply:
<img width="342" height="268" alt="Screenshot 2025-12-23 at 16 31 15" src="https://github.com/user-attachments/assets/8fcf94c5-b3a6-4eb8-9248-5dc7efd0aaec" />

Reply:
perhaps you don't need numbers at all, given its part of the layout already